src/components/profile/edit/EditLanguages.vue

```diff
@@ -6,22 +6,63 @@
     :initialValues="initialValues"
     formName="Edit languages"
   >
-    <p>Edit languages fields be here</p>
+    <header class="profile__section-header">
+      <h2>Languages</h2>
+    </header>
+    <Tag
+      v-for="(language, index) in languages.values"
+      :tag="language"
+      :key="`language-${index}`"
+      :removable="true"
+      @remove="removeLanguage(language)"
+    >
+    </Tag>
+    <input type="text" v-model="newLanguage" />
+    <button
+      type="button"
+      @click="
+        if (newLanguage.length > 0) {
+          addLanguage(newLanguage);
+        }
+      "
+    >
+      Add language
+    </button>
   </EditMutationWrapper>
 </template>
 
 <script>
+import Tag from '@/components/ui/Tag.vue';
+import { displayLevelsFor } from '@/assets/js/display-levels';
 import EditMutationWrapper from './EditMutationWrapper.vue';
 
 export default {
   name: 'EditLanguages',
   props: {
-    languages: Object,
     initialValues: Object,
     editVariables: Object,
   },
   components: {
     EditMutationWrapper,
+    Tag,
+  },
+  methods: {
+    displayLevelsFor,
+    addLanguage(language) {
+      const id = Object.keys(this.languages.values).length + 1;
+
+      this.languages.values[id] = language;
+      this.newLanguage = '';
```

Comment:
If it weren't for clearing `newLanguage` this wouldn't trigger a re-render. That's because Vue doesn't watch for deep object changes, so I'd change it accordingly. Multiple ways to do that, here's my favorite:
```javascript
this.languages = {...this.languages, [id]: language}
```

Recommended reading: https://vuejs.org/v2/guide/reactivity.html